draft-ietf-webpush-encryption.md

```diff
@@ -279,32 +279,25 @@ application server MUST set the `rs` parameter in the `aes128gcm` content coding
 header to a size that is greater than the length of the plaintext, plus any
 padding (which is at least 2 octets).
 
-A push message MUST include a zero length `keyid` parameter in the content
-coding header.
+A push message MUST include the application server ECDH public key in the
+`keyid` parameter of the encrypted content coding header.
```

Comment:
The subscription is encoded in the endpoint URL, 'from' is in the Authorization header.

It would have been nice if the binary format was extensible - I understand the motivation
of keeping it as simple as possible, but some tag/len or other mechanism are not that
complicated.